components/ModalDropdown.js

```diff
@@ -14,6 +14,8 @@ import {
   View,
   Text,
   ListView,
```

Comment:
Nice, using your fork since this repo seems like an orphan.

`ListView` import here should just be removed.